app/main.py

```diff
@@ -1,7 +1,17 @@
+import requests
+import os
+
+from dotenv import load_dotenv
+
+load_dotenv()
 def get_weather() -> None:
     # write your code here
-    pass
+    URL = "http://api.weatherapi.com/v1/current.json?"
+    FILTERING = "Paris"
+    API = os.getenv("API")
```

Comment:
Ensure that the environment variable 'API' is correctly set in your '.env' file. If it's not set, `os.getenv('API')` will return `None`, which will cause the API request to fail.